src/components/balance/FormatBalance.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>{{ formatBalance }} PLM</div>
```

Comment:
Please make the token symbol be dynamically parsed from the chain so that it displays PLD for Dusty, PLM for Plasm and other symbol for other metadata